README.md

```diff
@@ -1 +1,6 @@
 The decrypto.org website.
+
```

Comment:
Please add more detailed setup instructions. `yarn start` is not enough. I tried first installing yarn through my distro's package manager, then `npm install webpack-dev-server` with and without `sudo`, then `npm install webpack` with and without `sudo` and then the error was `Cannot find module 'webpack'` so I gave up.

Reply:
You need `yarn install` – we should add that.

Reply:
Full list of commands that made it work:
```sudo emerge yarn (or install yarn with your package manager)
sudo yarn install
sudo yarn add webpack
sudo yarn add webpack-dev-server
yarn start```

Reply:
Too many `sudo`s IMO...

Reply:
You shouldn't need `sudo` to run `yarn install`. You shouldn't need `sudo` to install `yarn` if you're using `npm` (but that's your choice based on your OS). Also `yarn add webpack` and `yarn add webpack-dev-server` should not be necessary. Can you try `yarn install --production=false`? Perhaps it's skipping devDependencies.

Reply:
1. Install [yarn](https://yarnpkg.com/lang/en/docs/install/#linux-tab)
2. Go inside repo folder i.e `~/workspace/decrypto.org`
3. `yarn install`
4. `yarn start`

This steps should be enough. You shouldn't need `sudo` to run `yarn` commands. Are you sure you have the fetch PR #6  and the `package.json` file is there?

We should add more details :)

Reply:
Probably `sudo` was needed because I installed `yarn` with `emerge`. The following worked without any `sudo`:
```
cd into the repo directory
npm install yarn
yarn install
yarn start
```

Reply:
This way is also cleaner because if I do `rm -rf decrypto.org` there is no trace of anything left on my machine.

Reply:
Done. I pass the `npm` step because of:

>Note: Installation of Yarn via npm is generally not recommended. When installing Yarn with Node-based package managers, the package is not signed, and the only integrity check performed is a basic SHA1 hash, which is a security risk when installing system-wide apps.

>For these reasons, it is highly recommended that you install Yarn through the installation method best suited to your operating system.

Also `npm install yarn` installs yarn locally, per project. It is better to install it globally (`npm install --global yarn`) if you are using it.

Reply:
But then a bunch of `sudo`s are needed, some of them also installing other things.

Reply:
I'm not sure which practice is better as a matter of fact...